sprintly/sprintly-ui
# Sortable table

#### What's this PR do?
Adds a SortableTable component, as well as item property components (Estimator, Tags, TagEditor, etc), and makes some organizational changes. Also implements new inline CSS.

#### Where should the reviewer start?
SortableTable/index.js

#### How should this be manually tested?
Two options: to view basic table, you may choose to load up the sortable_table page in examples locally. To see a fully sortable implementation, load up the sister branch in sprintly --> https://github.com/sprintly/sprint.ly/pull/1747

#### Any background context you want to provide?
This PR supports work to move sprintly over to cross-product, backed by separate data services and ui layers that we're making public.

#### What ticket or issue # does this fix?
Related sprintly ticket is #8607

#### Screenshots (if appropriate)
![screen shot 2015-01-15 at 2 55 55 pm](https://cloud.githubusercontent.com/assets/1137259/6068710/31f1d0f4-ad33-11e4-9528-e119fd9b8a1e.png)

#### What gif best describes this PR or how it makes you feel?
![giphy-14](https://cloud.githubusercontent.com/assets/1137259/6068716/41b17d3c-ad33-11e4-9889-53a77ee5224b.gif)

#### Definition of Done:
- [x] Does this contain breaking changes?
- [x] Did you bump the version number accordingly?
- [ ] Will this feature require a new piece of infrastructure be implemented?
- [x] Did you update the documentation per these changes?
